jekyll/_cci2/github-apps-integration.adoc

```diff
@@ -85,24 +85,9 @@ Project roles give control over which users have access to which projects within
 For a complete list of permissions, see the xref:roles-and-permissions-overview#[Roles and permissions overview] page.
 
 [#configuration]
-=== Configuration
+=== Configuration sources and project triggers
```

Comment:
It's a detail, but I would say just 
> Configuration sources and triggers
rather than "project" triggers

Reply:
@BeFunes I went with "project triggers" as that's how they are described in the UI, but maybe that will change. I'm happy to update if you think dropping the "project" is better

Reply:
parking this change to remove "project" for now